docs/components_and_bundles/bundles/SyliusResourceBundle/create_resource.rst

```diff
@@ -180,6 +180,28 @@ In addition to the request parameters, you can access some of the newly created
 
 With this configuration, the ``title`` parameter for route ``app_book_show`` will be obtained from your newly created book.
 
+Custom Event Name
+-----------------
+
+By default, there are two events thrown during resource creation, one before adding it do database, the other after successful addition.
```

Comment:
"thrown" -> "triggered"? Thrown sounds like an exception :)

Reply:
Maybe "dispatched"? Would be the most accurate, I think :)